examples/local/vttablet-up.sh

```diff
@@ -71,11 +76,27 @@ for uid_index in $uids; do
     echo "    $VTDATAROOT/$tablet_dir"
     action='start'
   fi
+
+  set +e
+
   $VTROOT/bin/mysqlctl \
     -log_dir $VTDATAROOT/tmp \
     -tablet_uid $uid \
     -mysql_port $mysql_port \
-    $action &
+    $action 
+
+    err=$?
+    if [[ $err -ne 0 ]]; then
+	echo "Checking for apparmor status for mysqld:"
+	sudo aa-status | grep mysqld
```

Comment:
Stock Ubuntu will pause and wait for a password here. It is only cloud instances that have sudo no password configured. We should instead recommend running this command to check.

Reply:
That is right.. @chrisrgithub you will need to remove this check but still print a helpful message.